lib/game.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  attr_reader :score, :scorecard
+
+  def initialize(frame1 = Frame.new, frame2 = Frame.new,
+                 frame3 = Frame.new, frame4 = Frame.new,
+                 frame5 = Frame.new, frame6 = Frame.new,
+                 frame7 = Frame.new, frame8 = Frame.new,
+                 frame9 = Frame.new, frame10 = Frame.new)
+    @score = 0
+    @scorecard = { frame1: frame1, frame2: frame2,
+                   frame3: frame3, frame4: frame4,
+                   frame5: frame5, frame6: frame6,
+                   frame7: frame7, frame8: frame8,
+                   frame9: frame9, frame10: frame10 }
+  end
+
+  # this method accepts an integer input from the user.
+  # it checks what the current frame is
+  # it updates the framescore for that frame appropriately
+  def input_score(score)
```

Comment:
This is the only method in your code that I would think about refactoring to reduce its length / complexity